linkerd/announcer/serversets/src/main/scala/com/twitter/finagle/zookeeper/buoyant/ZkAnnouncer.scala

```diff
@@ -0,0 +1,15 @@
+package com.twitter.finagle.zookeeper.buoyant
+
+import com.twitter.finagle.{Announcement, Announcer}
+import com.twitter.finagle.zookeeper.{ZkAnnouncer => FZkAnnouncer, DefaultZkClientFactory}
+import com.twitter.util.Future
+import java.net.InetSocketAddress
+
+class ZkAnnouncer(hosts: String, pathPrefix: String) extends Announcer {
+  override val scheme: String = "zk-serversets"
+
+  val underlying = new FZkAnnouncer(DefaultZkClientFactory)
+
+  override def announce(addr: InetSocketAddress, name: String): Future[Announcement] =
+    underlying.announce(addr, s"$hosts!$pathPrefix/$name!0")
```

Comment:
is this really a name? it seems in many cases this will be a path like role/env/job -- we should probably be graceful (or at least error explicitly) about leading slashes